src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportGenerator;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String generateReport(Map<String, Integer> inventory) {
+        StringBuilder report = new StringBuilder("fruit,quantity\n");
+        inventory.forEach((fruit, quantity) ->
+                report.append(fruit).append(",").append(quantity).append("\n")
+        );
+        return report.toString();
+    }
```

Comment:
The class 'ReportGeneratorImpl' should not begin with an empty line according to the checklist. Please remove the empty line after the class declaration to adhere to the coding standards.